roles/datadog-builder/tasks/datadog-builder.yml

```diff
@@ -64,3 +73,4 @@
       /opt/venvs/datadog-builder/bin/datadog-builder
       --config /etc/datadog-builder/datadog-secrets.yml
       update {{ datadog_builder_monitor_dir }}/{{ datadog_builder_monitor_file }}
+      2>> /var/log/datadog-builder.log > /dev/null
```

Comment:
I redirect stderr to stdout now, just in case we start getting useful things from stdout in the future